src-tauri/src/main.rs

```diff
@@ -19,12 +22,12 @@ fn main() {
     let context = tauri::generate_context!();
 
     tauri::Builder::default()
-        .setup(|app| {
+        .setup(|_app| {
```

Comment:
Why do we need to rename this?

Reply:
That's a weird warning during `yarn tauri build` for unused variable 😕.
I didn't spend time understanding what is going on (is not that important for now, IMO), so just followed the compiler instructions to disable the warning.